privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -849,6 +853,7 @@ def loadConfig(self, config):
         self.serverpool_persistent = is_true(config.get("SERVERPOOL_PERSISTENT", False))
         self.serverpool_rounds = int(config.get("SERVERPOOL_ROUNDS") or SERVERPOOL_ROUNDS)
         self.serverpool_skip = int(config.get("SERVERPOOL_SKIP") or SERVERPOOL_SKIP)
+        self.serverpool_strategy = config.get("SERVERPOOL_STRATEGY") or SERVERPOOL_STRATEGY
```

Comment:
Please also add this parameter to the `testconnection()` function:
https://github.com/privacyidea/privacyidea/blob/b2bf17b8097e5c98a10c43e087e8983626011ed6/privacyidea/lib/resolvers/LDAPIdResolver.py#L1044